09-features.md

```diff
@@ -19,6 +19,7 @@ These flags may only be used in the `init` message:
 | 0/1  | `option-data-loss-protect` | Requires or supports extra `channel_reestablish` fields | [BOLT #2](02-peer-protocol.md#message-retransmission) |
 | 3  | `initial_routing_sync` | Indicates that the sending node needs a complete routing information dump | [BOLT #7](07-routing-gossip.md#initial-sync) |
 | 4/5  | `option_upfront_shutdown_script` | Commits to a shutdown scriptpubkey when opening | [BOLT #2](02-peer-protocol.md#the-open_channel-message) |
+| 8/9  | `option-funding-cancelled` | Requires or supports `funding_cancelled` message | [BOLT #2](02-peer-protocol.md#the-funding_cancelled-message) |
```

Comment:
Nit: Doesn't "require" imply "support" in this context? Would `Supports funding_cancelled message` be enough?

Reply:
Even bit requires the feature, odd bit supports if possible ("its ok to be odd").

Reply:
See also bits 0/1

Reply:
@ZmnSCPxj Ah, my bad! Missed the `8/9`. Sorry! :-)